mmselfsup/models/algorithms/maskfeat.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List, Optional
+
+import torch
+
+from ..builder import ALGORITHMS, build_backbone, build_head
+from ..utils.hog_layer import HOGLayerC
+from .base import BaseModel
+
+
+@ALGORITHMS.register_module()
+class MaskFeat(BaseModel):
+    """MaskFeat.
+
+    Implementation of `Masked Feature Prediction for
+    Self-Supervised Visual Pre-Training <https://arxiv.org/abs/2112.09133>`_.
+    Args:
+        backbone (dict): Config dict for encoder. Defaults to None.
```

Comment:
no default value of `backbone`, `head`, `hog_para`
the docstring of `Defaults to None` can be removed